packages/pages/src/build/build.ts

```diff
@@ -0,0 +1,26 @@
+import shell from "shelljs";
+import { CommandModule } from "yargs";
+import { ProjectFilepaths } from "../common/src/project/structure.js";
+
+type BuildArgs = Pick<ProjectFilepaths, "scope">;
+
+const handler = async ({ scope }: BuildArgs) => {
+  // Pass CLI arguments as env variables to use in vite-plugin
+  if (scope) {
+    process.env.YEXT_PAGES_SCOPE = scope;
+  }
+  shell.exec("vite build");
```

Comment:
child_process.spawn is used in a few other places, would we want to use instead there as well? If we did I was thinking we could add it to the doc

Reply:
sure! What do you mean add it to the doc?

Reply:
sure! what do you mean by add it to the doc?

Reply:
oh the pagesjs doc with the assorted small cleanup things we'd want